data/mods/Boats/b_wooden_boat_parts.json

```diff
@@ -3,6 +3,7 @@
     "type" : "vehicle_part",
     "id" : "boat_board",
     "name" : "boat board",
+	"description": "A wooden board that keeps the boat afloat.  To change a vehicle into a boat, place as you would wheels on a car. Then attach oars or a motor to get the boat to move.",
```

Comment:
Sadly, this is not recognized by the game because vehicle parts don't have descriptions. But I suggest to leave it in for later in case they ever get them.